signer-wasm/src/ledger.rs

```diff
@@ -89,30 +109,17 @@ pub async fn key_retrieve_from_device(
     let apdu_transport = APDUTransport {
         transport_wrapper: tmp,
     };
-
     let app = filecoin_signer_ledger::app::FilecoinApp::new(apdu_transport);
-
     // FIXME: reconcile BIP44Path different implementation
```

Comment:
I know it is not part of this PR, but I think we should create an issue for this and remove the comment

Reply:
Not sure what "different implementation" means. Would it be related to different types of inputs like a string or a collection of bytes?

Reply:
This has been fixed actually so the comment can just be removed. It was because at one point we had 2 Bip44Path structure doing the same thing but coming from 2 different codebase. I have seen put everything in `ledger-rs`.

Reply:
Thanks!